mbedtls/src/ssl/config.rs

```diff
@@ -329,6 +329,49 @@ impl<'c> Config<'c> {
             )
         }
     }
+
+    /// psk and psk_identity cannot be empty
+    pub fn set_psk(&mut self, psk: &[u8], psk_identity: &str) -> Result<()> {
+        assert!(psk_identity.len()>0);
+        assert!(psk.len()>0);
```

Comment:
The `mbedtls_ssl_conf_psk` function already checks these arguments. I don't see a reason to check them here again